utils/query.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import "net/http"
+
+func GetResponseDataFromQuery(r *http.Request) (ResponseData, error) {
```

Comment:
As an improvement, there could be just one parser that could handle multiple response modes.
The logic should check if there is a response in query/fragment when parse jwt. Otherwise parse query / fragment. 
There is also form_post mode that we could add at some point.